### PR TITLE
Update bun.mdx

### PR DESCRIPTION
**Pull request by Harold-Anderson**
Used bun to build Astro 5 site on Cloudflare Pages.  Discussion of changes needed and improvement in build time.

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Review by at-the-vr — COMMENTED** [review]
There's not really a quality check to moderate community content besides making sure the content is not outdated but I would really love to see this article possibly in your personal blog (README's are difficult)

**Comment by Harold-Anderson** [comment]
@at-the-vr  Would you prefer to see it like this https://handerson.hashnode.dev/using-bun-with-astro-and-cloudflare-pages ?
If that is acceptable, I'll clean up the personal blog article and try to keep the content from the README and the blog in sync somehow.

File: src/content/docs/en/recipes/bun.mdx [hunk 1]
```diff
@@ -135,3 +135,4 @@ To create your first Playwright test, follow the instructions for the rest of th
 Using Bun with Astro? Add your blog post or video to this page!
 
 - [Building a Cloudflare Pages site with Bun](https://blog.otterlord.dev/posts/hello-from-bun/) - blog post
+- [Comparing Bun as a build tool to pnpm for Cloudflare Pages](https://handerson.hashnode.dev/using-bun-with-astro-and-cloudflare-pages) -- blog post
```
Comment (at-the-vr) [inline src/content/docs/en/recipes/bun.mdx:138]: I think this is way more accurate now from previous. What do you say?
```suggestion
- [Using Bun with Astro and Cloudflare Pages](https://handerson.hashnode.dev/using-bun-with-astro-and-cloudflare-pages) - blog post
```

**Review by at-the-vr — APPROVED** [review]
Thank you so much Harold. This article is really way better to read now. As a suggestion you can prevent duplicity by keeping the hashnode article up to date and link it in your repo's README but whatever method works out for ya

**Review by TheOtterlord — APPROVED** [review]
Thanks for updating this @Harold-Anderson!

**Pull request merged**